_includes/card-purchase.md

```diff
@@ -260,6 +260,12 @@ Content-Type: application/json
             "method": "GET",
             "contentType": "application/javascript"
         }
+        {
+            "rel": "direct-authorization",
+            "href": ""{{ page.front_end_url }}/creditcard/payments/{{ page.payment_token }}/authorizations",
```

Comment:
I don't think the `direct-authorization` operation should be performed against the `front_end_url`.

```suggestion
            "href": ""{{ page.api_url }}/psp/creditcard/confined/payments/{{ page.payment_id }}/authorizations",
```